prow/config.yaml

```diff
@@ -20271,6 +20271,47 @@ periodics:
       secret:
         secretName: velodrome-influxdb
 
+- interval: 24h
```

Comment:
do you care what time of day you want the job to run? If so make a `cron:` entry rather than use interval.

Reply:
Oh, cool, thanks! I should have searched for `cron`. Fixed.

Reply:
@krzyzacy Seems like cron is finding use by other folks too :)

Reply:
yeah I'm doing hard sell :-)